local-kubernetes/local-isapi-sample-kubernetes-manifest.yml

```diff
@@ -0,0 +1,166 @@
+apiVersion: apps/v1
```

Comment:
same as above comment - why include this?

Reply:
as above replies.